stack/variable.go

```diff
@@ -12,6 +12,8 @@ type Variable struct {
 	Description string
 	Type        cty.Type
 
+	IsSensitive bool
```

Comment:
I think we need to set this attribute in the early decoder or else it will never get a value

Reply:
`!false` 😄

Reply:
done!